main.tf

```diff
@@ -186,3 +197,76 @@ resource "google_storage_bucket" "phs-temp-bucket" {
   uniform_bucket_level_access = true
   force_destroy               = var.force_destroy
 }
+
+resource "google_storage_bucket" "sparkml-model-bucket" {
+  name                        = "gcp-${var.use_case_short}-model-${random_id.id.hex}"
+  project                     = module.project-services.project_id
+  location                    = var.region
+  uniform_bucket_level_access = true
+  force_destroy               = var.force_destroy
+}
+
+# Creates a service account specifically for the Workbench instance.
+resource "google_service_account" "workbench_service_account" {
```

Comment:
Can you move the Workbench resource and the two SA resources to a separate `workbench.tf` file?